website/client/components/static/staticWrapper.vue

```diff
@@ -0,0 +1,88 @@
+<template lang='pug'>
+  div
+    static-header
+
+    app-page(v-if='$route.name === "app"')
+    clear-browser-data(v-if='$route.name === "clearBrowserData"')
+    community-guidelines(v-if='$route.name === "communityGuidelines"')
+    contact-page(v-if='$route.name === "contact"')
+    faq-page(v-if='$route.name === "faq"')
+    features-page(v-if='$route.name === "features"')
+    group-plans(v-if='$route.name === "groupPlans"')
+    home-page(v-if='$route.name === "home"')
+    merch-page(v-if='$route.name === "merch"')
+    overview-page(v-if='$route.name === "overview"')
+    press-kit(v-if='$route.name === "pressKit"')
+    privacy-page(v-if='$route.name === "privacy"')
+    terms-of-use(v-if='$route.name === "terms"')
+
+    app-footer
+</template>
+
+<style lang='scss'>
```

Comment:
this should be `scoped` but probably won't work when you include child components so I'd say give a class `.static-wrapper` to the root div and put all the css rules under it like

```
.static-wrapper {
    h1, h2 { ... }
}
```

Otherwise this rules will become global and could affect all pages of the site